style.css

```diff
@@ -0,0 +1,353 @@
+  align-items: center;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  overflow-x: hidden;
+}
+
+.hamburger {
+  cursor: none;
+  margin-right: 10%;
+}
+
+.home-body {
+  background-image: url(Assets/files/header\ shapes.png);
+  background-repeat: no-repeat;
+  min-width: 60%;
+  height: 604px;
+}
+
+.heading {
+  min-width: 300px;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  margin-left: 1.3rem;
+  padding-top: 6.5rem;
+}
+
+.paragraph {
+  min-width: 50%;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 1.3rem;
+  margin-top: -1.7rem;
+}
+
+.connect {
+  color: #7f8cff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 1.3rem;
+}
+
+#social-links {
+  display: flex;
+  padding-right: 11.7rem;
+  padding-top: 2%;
+  margin-left: -1.2rem;
+  justify-content: space-between;
+  list-style: none;
+  margin-top: 0.5%;
+}
+
+#works {
+  display: grid;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  column-gap: 5px;
+  list-style: none;
+}
+
+.container1 {
+  background: white;
+  height: 552px;
+  width: 100%;
+  border-radius: 16px;
+  margin-top: 12%;
+  margin-left: -10%;
+}
+
+.container2 {
+  background: white;
+  height: 596px;
+  width: 100%;
+  border-radius: 16px;
+  margin-top: 13%;
+  margin-left: -10%;
+}
+
+.works-pic {
+  margin-top: 5%;
+  margin-left: -7%;
+}
+
+.works-heading {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: rgba(23, 43, 77, 1);
+  margin-left: -6%;
+  padding-top: 2%;
+  width: 200px;
+}
+
+.works-paragraph {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  height: 76px;
+  max-width: 280px;
+  top: 92px;
+  margin-left: -5%;
+  color: rgba(52, 69, 99, 1);
+}
+
+.canopy {
+  margin-left: -5%;
+}
+
+.languages {
+  display: flex;
+  gap: 1rem;
+  margin-top: 10px;
+  margin-left: -3px;
+}
+
+.languages li {
+  padding: 1px 8px;
+  background: #eeeef7;
+  font-size: 15px;
+  border-radius: 8px;
+  border: 0;
+  color: rgba(96, 112, 255, 1);
+}
+
+.languages li:nth-child(1) {
+  margin-left: -3.4rem;
+}
+
+button {
+  margin-left: -5%;
+  margin-top: 7%;
+  background-color: #fcfbfe;
+  padding: 10px;
+  border: 1px solid rgba(57, 109, 242, 1);
+  border-radius: 16px;
+  color: rgb(86, 135, 235);
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+}
+
+button:hover {
+  background-color: #7f8cff;
+  color: #dfe1e6;
+}
+
+#about-self {
+  background-image: url(Assets/files/about\ myself\ -\ background-\ shape-\ desktop.jpg);
+  border-top-right-radius: 40px;
+  margin-left: -0.5rem;
+  margin-top: 6rem;
+  position: absolute;
+  height: 1263px;
+}
```

Comment:
- Good job done so far, but according to the project requirements, please use the `display:flex` property to align the contents of this section. Doing this will help with the responsiveness of this section 👍 👍